src/service/viewport-impl.js

```diff
@@ -750,6 +750,10 @@ export class ViewportBindingNatural_ {
     if (this.win.document.defaultView) {
       waitForBody(this.win.document, () => {
         this.win.document.body.style.overflow = 'visible';
+        setStyles(this.win.document.body, {
+          overflowX: 'hidden',
+          overflowY: 'auto',
+        });
```

Comment:
> Only do any customization when #webview=1 is specified.

This problem also happens outside webview in any WebKit browser when the article is not embedded. Do we not care about horizontal scrolling happening there?

> We definitely, under no circumstances, can have overflow-y: auto on body in this mode.

Why?